framework-plugins/lisk-framework-http-api-plugin/src/controllers/node.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { Request, Response, NextFunction } from 'express';
+import { BaseChannel } from 'lisk-framework';
+
+export const getNodeStatusAndConstants = (channel: BaseChannel) => async (
+	_req: Request,
+	res: Response,
+	next: NextFunction,
+): Promise<void> => {
+	try {
+		const nodeStatusAndInfo: Buffer = await channel.invoke('app:getNodeStatusAndConstants');
```

Comment:
bit confusing, `nodeStatusAndInfo` is json object? if its buffer then below should be decoded `send(nodeStatusAndInfo)`?

Reply:
Fixed